src/WinRT.Runtime/ObjectReference.cs

```diff
@@ -491,12 +516,38 @@ public static unsafe ObjectReference<T> FromAbi(IntPtr thisPtr, T vftblT)
             {
                 var obj = new ObjectReferenceWithContext<T>(
                     thisPtr,
+                    vftblT,
                     Context.GetContextCallback(),
                     Context.GetContextToken());
                 return obj;
             }
         }
 
+        public static unsafe ObjectReference<T> FromAbi(IntPtr thisPtr, T vftblT, Guid iid)
```

Comment:
Is this needed? Doesn't seem like anyone is actually calling it?

Reply:
Called by other FromAbi methods.